# missing footnote for 3)
# figured out what footnote was missing by comparing the following two pages:
# - https://www.althingi.is/lagas/151c/2004081.html
# - https://www.althingi.is/lagas/151b/2004081.html
--- data/cleaned/2004-81.html	2022-02-19 20:11:44.278432635 +0000
+++ data/patched/2004-81.html	2022-02-19 20:11:44.282432632 +0000
@@ -247,20 +247,26 @@
     </sup>
     <a href="/altext/stjt/2021.053.html">
      L. 53/2021, 1. gr.
     </a>
     <sup style="font-size:60%">
      2)
     </sup>
     <a href="/altext/stjt/2020.085.html">
      L. 85/2020, 6. gr.
     </a>
+    <sup style="font-size:60%">
+     3)
+    </sup>
+    <a href="/altext/stjt/2011.126.html">
+     L. 126/2011, 389. gr.
+    </a>
    </small>
   </i>
   <br/>
   <span id="G3">
   </span>
   <img alt="" height="11" src="sk.jpg" width="11"/>
   <b>
    3. gr.
   </b>
   <em>
